partner_tlr_update/models/res_partner.py

```diff
@@ -256,7 +261,7 @@ def update_sub_address(self, xml, performing_operation_id):
             if contact_persons is not None:
                 _logger.info("CONTACT PERSONS NOT NONE")
                 for elem in contact_persons:
-                    self.update_contact_person(elem, partner.id)
+                    self.update_contact_person(elem, partner.id, partner.id)
```

Comment:
Should this this really take in partner.id twice? The second argument should be an address?

Reply:
partner.id in this case is the id of the address partner. I needed to do it this way s.ince only some contact persons are created from here and I need a way to set both the parent_id of the contact person partner and the address_id of the contact person employee, which is False when not created from here.